tests/test_minio_adapter.py

```diff
@@ -160,6 +160,25 @@ async def test_download_short_filename_change(minio_adapter, populate_bucket):
     result.unlink()  # it should exist, from above ...
 
 
+@pytest.mark.parametrize("populate_bucket", ["test.txt"], indirect=True)
+@pytest.mark.asyncio
+async def test_download_repeat(minio_adapter, populate_bucket):
+    import asyncio
+
+    print(await minio_adapter.list_bucket())
+    result = await minio_adapter.download_file("test.txt", local_dir="/tmp/foo")
```

Comment:
You parameterize this test with populate_bucket as test.txt, but have it hardcoded here too.

Reply:
The parametrization parametrizes the argument to the fixture `populate_bucket`; I'm not aware of a mechanism to make it available to the test function itself (other than doing an additional parametrization with the same argument). In general since we have a fair number of cases where the two arguments are different I don't know that it's super useful to do that, the values should be pretty clear.